formula-android/src/main/java/com/instacart/formula/android/FragmentState.kt

```diff
@@ -10,8 +10,8 @@ package com.instacart.formula.android
 data class FragmentState(
     val activeIds: List<FragmentId> = emptyList(),
     val visibleIds: List<FragmentId> = emptyList(),
-    val features: Map<FragmentId, FeatureEvent> = emptyMap(),
-    val outputs: Map<FragmentId, FragmentOutput> = emptyMap()
+    val outputs: Map<FragmentId, FragmentOutput> = emptyMap(),
+    internal val features: Map<FragmentId, FeatureEvent> = emptyMap(),
```

Comment:
This is an internal variable used by formula, likely will remove it in the future.